src/lib/Initialization.js

```diff
@@ -33,9 +36,17 @@ export function getTestGroupDetails() {
     return IronWeb.group.get(existingTestGroup);
 }
 
+
 /**
  * Initialize the IronWeb SDK as the provided user ID
  */
 export function initializeAsUser(userID) {
     return IronWeb.initialize(() => getJWT(userID), () => getUserPasscode());
 }
+
+/**
+ * Check if IronWeb SDK is initialized and return a boolean value indicating whether it is or not.
+ */
+export function returnIsInitialized() {
+    return IronWeb.isInitialized();
+}
```

Comment:
Why not just call this method directly? Is there a large benefit to having a wrapper function around this?